content/applications/finance/accounting/taxes.rst

```diff
@@ -4,167 +4,219 @@
+2. Odoo automatically computes tax amounts on the documents.
+3. On accounting documents, Odoo generates journal items to keep track of tax debit and tax credit.
+4. The total base, tax debit and tax credit for a period can be viewed in the Tax Return report
+   and used to file a tax return.
 
-.. image:: taxes/default-configuration.png
-   :alt: Odoo fills out the Tax field automatically according to the Default Taxes
+Taxes on sales and purchases
+----------------------------
 
-To change your **default taxes**, go to :menuselection:`Accounting --> Configuration --> Settings
---> Taxes --> Default Taxes`, select the appropriate taxes for your default sales tax and purchase
-tax, and click on :guilabel:`Save`.
+Most sale and purchase documents have a :guilabel:`Taxes` field where taxes can be applied to
+individual lines. This includes :doc:`invoices <customer_invoices>` and :doc:`vendor bills
+<vendor_bills>` in the :guilabel:`Accounting` app, :doc:`sales quotations
+<../../sales/sales/sales_quotations>` in the :guilabel:`Sales` app, and :doc:`purchase orders
+<../../inventory_and_mrp/purchase/manage_deals/rfq>` in the :guilabel:`Purchase` app.
 
-.. image:: taxes/default-taxes.png
-   :alt: Define which taxes to use by default on Odoo
+.. image:: taxes/invoice-tax.png
+   :alt: Adding a 21% tax on an invoice line.
+
+.. tip::
+   When adding a product to a sale or purchase line, the taxes :ref:`assigned to the product
+   <taxes/product>` are automatically applied on the line.
 
 .. note::
-   **Default taxes** are automatically set up according to the country selected at the creation of
-   your database, or when you set up a :ref:`fiscal localization package
-   <fiscal_localizations/packages>` for your company.
+   In the :doc:`Point of Sale <../../sales/point_of_sale>`, the taxes :ref:`assigned to the products
+   <taxes/product>` are applied directly to the order and cannot be changed except by a
+   :doc:`fiscal position <taxes/fiscal_positions>`.
 
-.. _taxes/list_activation:
+Automatic computation of tax amounts
+------------------------------------
+
+Applying a tax on a sale or purchase line lets Odoo automatically compute a tax amount based on
+the sale or purchase line's subtotal and the tax's configuration. The details of the computation are
+explained in the :doc:`Tax Computation <taxes/tax_computation>` page.
+
+.. image:: taxes/invoice-tax-amount.png
+   :alt: Odoo automatically computes a tax amount for the 21% tax.
+
+Automatic generation of tax journal items
+-----------------------------------------
+
+Upon applying a tax on an :doc:`Invoice <customer_invoices>` or :doc:`Vendor Bill <vendor_bills>`, a
+tax payable journal item is automatically generated with the tax amount. This keeps track of the tax
+debit or credit associated with the transaction.
 
-Activate taxes from the list view
-=================================
+Furthermore, the tax amount is added to the amount due on the Account Receivable or Payable journal
```

Comment:
```suggestion
Furthermore, the tax amount is added to the amount due on the account receivable or payable journal
```
Maybe use italics instead of capitals to distinguish these specific terms?